package/gluon-config-mode-geo-location/src/de.po.in

```diff
@@ -0,0 +1,54 @@
+msgid ""
+msgstr ""
+#ifdef WITHMAP
+"Project-Id-Version: gluon-config-mode-geo-location-with-map\n"
+#else
+"Project-Id-Version: gluon-config-mode-geo-location\n"
+#endif
+"PO-Revision-Date: 2017-08-22 12:14+0100\n"
+"Last-Translator: Jan-Tarek Butt <tarek@ring0.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"If you want the location of your node to be displayed on the map, you can "
+"enter its coordinates here. "
+#ifdef WITHMAP
+"If your PC is connected to the internet you may also click on the map displayed below. "
+#endif
+"Please keep in mind setting a location can also enhance the network quality."
+msgstr ""
+"Um deinen Router auf der Karte anzeigen zu können, benötigen wir seine "
+"Koordinaten. Hier hast du die Möglichkeit, "
```

Comment:
Position

Reply:
I think it should be "Koordinaten" which describes the nodes "Position"

Reply:
Talking about a position and then asking for coordinates for that position sounds more wholesome to me, that's why I recommended it that way.